build.sbt

```diff
@@ -17,7 +17,7 @@ val AwsSdkVersion = "2.9.1"
 val Fs2Version = "2.4.5"
 val MonixVersion = "3.3.0"
 // Make sure this is the same for different sub projects
-val NettyVersion = "4.1.49.Final"
+val NettyVersion = "4.1.52.Final"
```

Comment:
Is this just documentation? It isn't used anywhere I can see.

Reply:
Ah yes, I noticed this too, was just busy removing this